templates/medical/blood_donation.txt

```diff
@@ -0,0 +1,46 @@
+[center][b]Blood Donation Application Form[/b][/center]
```

Comment:
Would rephrase to "blood donation informed consent form" or something, "application" isn't quite the word for the process imo